commons-geometry-euclidean/src/main/java/org/apache/commons/geometry/euclidean/threed/Bounds3D.java

```diff
@@ -233,7 +233,7 @@ public Builder add(final Vector3D pt) {
          * @param pts points to add
          * @return this instance
          */
-        public Builder addAll(final Iterable<Vector3D> pts) {
+        public Builder addAll(final Iterable<? extends Vector3D> pts) {
```

Comment:
Is this worth it? The only subclass of `Vector3D` is `Vector3D.Unit`.

Reply:
Although there is one subclass within the library, users can define other subclasses of `Vector3D` (for whatever reason).

Reply:
Users can't create their own `Vector3D` subclasses since the constructor is private. I don't see a reason to prevent use of `Iterable<Vector3D.Unit>`, though, so I think we should go ahead with this change.